src/diffusers/pipelines/wuerstchen/modules.py

```diff
@@ -0,0 +1,389 @@
+import math
+
+import numpy as np
+import torch
+import torch.nn as nn
+
+from diffusers.configuration_utils import ConfigMixin, register_to_config
+
+from ...models.modeling_utils import ModelMixin
+
+
+class LayerNorm2d(nn.LayerNorm):
```

Comment:
As stated before we should not create a layer norm abstraction here but just do:
```py
(x.permute(0, 2, 3, 1)).permute(0, 3, 1, 2)
```
directly in the code